src/Orchard.Web/Modules/Orchard.OpenId/Controllers/AccountController.cs

```diff
@@ -46,6 +46,7 @@ public class AccountController : Controller {
         public ILogger Logger { get; set; }
         public Localizer T { get; set; }
 
+        [AlwaysAccessible]
```

Comment:
I think that this is not required, the whole controller is always accessible by default.